packages/openapi-fetch/test/index.test.ts

```diff
@@ -605,12 +605,13 @@ describe("client", () => {
       it("stream", async () => {
         const client = createClient<paths>();
         mockFetchOnce({ status: 200, body: "{}" });
-        const { data } = await client.GET("/anyMethod", {
+        const { data } = (await client.GET("/anyMethod", {
           parseAs: "stream",
-        });
+        })) satisfies { data?: ReadableStream<Uint8Array> | null };
         if (!data) {
           throw new Error(`parseAs stream: error`);
         }
+        // todo - not sure what test to put here - previously it just checked for instanceof Buffer
         expect(data.byteLength).toBe(8);
```

Comment:
Not sure what test to put here - previously it just checked for instanceof Buffer

Reply:
Oh that can change to whatever. I just wanted to have a type test because usually TS will let you check for `instanceof` on anything (that’s a runtime test). Really the `expect()` here is useless; it’s more for the test reader it’s an assertion. But any check will work where we try and access something off the `data` type immediately without guiding the inference first (which `instanceof` will do)

Reply:
@drwpow There's also this problem which I'm not sure we even need to address
https://github.com/node-fetch/node-fetch/issues/1308

The reported type from node's fetch(they use undici I think but I assume it's the same issue as with node-fetch) is ReadableStream as per browser spec but in nodejs it could either be a stream or a buffer.

Reply:
My proposed solution would be this
https://github.com/drwpow/openapi-typescript/pull/1428/commits/762efcab7be08753ea6765eff5b3b48954382da9

Reply:
Ah I see what you mean. Yeah, I agree with your approach. In general, this isn’t the only place where Node.js and browser types diverge, so the less work we can do here the better. This is also another advantage of us manually owning the types now—sometimes when building a library, your TSConfig settings change the generated types and you accidentally bake in Node.js types for a browser package or vice-versa. But now, there are no type surprises, but even if there are we can patch it without affecting runtime code.